README.md

```diff
@@ -24,6 +24,19 @@ cf set-env $YOUR_APP_NAME DD_API_KEY $YOUR_DATADOG_API_KEY
 cf restage $YOUR_APP_NAME
 ```
 
+You can also enable log collection, you will need to provide a TCP port to forward logs to the agent:
+```shell
+# set the environment variables
+cf set-env $YOUR_APP_NAME DD_API_KEY $YOUR_DATADOG_API_KEY
+cf set-env $YOUR_APP_NAME RUN_PUPPY true
+cf set-env $YOUR_APP_NAME DD_LOGS_ENABLED true
+cf set-env $YOUR_APP_NAME DD_LOGS_CONFIG_TCP_FORWARD_PORT 10514
+# prevent collecting all logs from stdout
```

Comment:
could we improve the comment? Like explain what is the default behavior and how setting it to false will change it?